src/nvapi_private.h

```diff
@@ -49,3 +49,7 @@
 #elif defined(_MSC_VER)
 #include <intrin.h>
 #endif
+
+// clang-format off
+#define MAP_ENUM_VALUE(v) {v, #v}
```

Comment:
There is probably a better name..

Reply:
I think it's fine.

However, `// clang-format off` above looks suspicious. Does autoformatting break this horribly somehow?

Reply:
I added this because of https://github.com/jp7677/dxvk-nvapi/actions/runs/9650777826/job/26617157512
Unfortunately I didn’t hit this locally, so I took this shortcut.

Reply:
Ah, weird. We might be suffering from infamous `clang-format` version mismatch, although suppressing it for one `#define` is not a big deal.